src/Libraries/CoreNodes/Properties/Resources.en-US.resx

```diff
@@ -139,7 +139,7 @@
     <value>Wednesday</value>
   </data>
   <data name="ExportToCSVObsolete" xml:space="preserve">
-    <value>Use ImportExport.ExportCSV node instead</value>
+    <value>Use Data.ExportCSV node instead</value>
```

Comment:
I think these should remain as `Data.ExportCSV`? Since from the user's perspective the node exists as `Data.ExportCSV` and not `DSCore.IO.Data.ExportCSV`.